std_msgs/Header header
float64 depth                # depth reading
float64 depth_variance       # 0 is interpreted as variance unknown


